### PR TITLE
docs: added experiment overview

### PR DESCRIPTION
**Pull request by musslick**
# Description

Added an overview describing the experimentalists.

# Type of change
- **docs**: Documentation only changes

### CODE REVIEW

File: docs/experimentalists/overview.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Experimentalist Overview
+
+The primary goal of an experimentalist is to identify experiments that yield 
+scientific merit. AutoRA implements techniques for automating the identification 
+of novel experiments.
+
+An experiment consists of a series of **experiment conditions** $\vec{x} \in X$. 
+The experimental variables manipulated in each experiment condition 
+are defined as factors, and the values of each variable to be sampled
```
Comment (benwandrew) [inline docs/experimentalists/overview.md:9]:
```suggestion
are defined as **factors**, and the values of each variable to be sampled 
```

File: docs/experimentalists/overview.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+An experiment consists of a series of **experiment conditions** $\vec{x} \in X$. 
+The experimental variables manipulated in each experiment condition 
+are defined as factors, and the values of each variable to be sampled 
+in the experiment are defined as **levels** of the corresponding **factors**. 
+As an example, consider a visual discrimination tasks in which participants are presented
+with two lines of different lengths, and are asked to indicate which line is longer.
+There are two factors in this experiment: the length of the first line and 
+the length of the second line. Instances of the two line lengths 
+(e.g., 2.0 cm for the first line and 2.1 cm for the sceond line) 
+can be considered levels of the two factors, respectively. Thus, *an experiment condition is a vector of values,
```
Comment (benwandrew) [inline docs/experimentalists/overview.md:16]:
```suggestion
can be considered levels of the two factors, respectively. Thus, *an experiment condition is a vector of values that
```

File: docs/experimentalists/overview.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Experimentalists in AutoRA serve to identify novel 
+experiment conditions $\vec{x} \in X$, where $x_i$ corresponds 
+to the level of an experimental factor $i$.
+
+![Overview](../img/experimentalist.png)
+
+Experimentalists may use information about candidate models $M$ obtained from a theorist, 
+experiment conditions that have already been probed $\vec{x}' \in X'$, or 
+respective dependent measures $\vec{y}' \in Y'$. The following table the current implementations
```
Comment (benwandrew) [inline docs/experimentalists/overview.md:28]:
```suggestion
respective dependent measures $\vec{y}' \in Y'$. The following table includes the experimentalists currently implemented
```

File: docs/experimentalists/overview.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Experimentalists in AutoRA serve to identify novel 
+experiment conditions $\vec{x} \in X$, where $x_i$ corresponds 
+to the level of an experimental factor $i$.
+
+![Overview](../img/experimentalist.png)
+
+Experimentalists may use information about candidate models $M$ obtained from a theorist, 
+experiment conditions that have already been probed $\vec{x}' \in X'$, or 
+respective dependent measures $\vec{y}' \in Y'$. The following table the current implementations 
+of experimentalists in AutoRA.
```
Comment (benwandrew) [inline docs/experimentalists/overview.md:29]:
```suggestion
 in AutoRA.
```

**Review by benwandrew — CHANGES_REQUESTED** [review]
nice! i left a few suggestions. looks like we'll also need to resolve the failing checks...

**Comment by musslick** [comment]
> nice! i left a few suggestions. looks like we'll also need to resolve the failing checks...

Thanks a lot! Not sure why the tests are failing... Perhaps @hollandjg has an idea?

**Comment by benwandrew** [comment]
looks like they resolved themselves :)

i just resolved a merge conflict, so hopefully ready to go after the checks pass again (i hope!)

**Review by benwandrew — APPROVED** [review]

**Pull request merged**